src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -26,14 +26,6 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
-    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
-  },
-  {
```

Comment:
I'd rather have it show 7/5 than not at all; it's important viewers of the analyzer become aware that they have that spell and should use it, even if it's not in the suggestions.